### PR TITLE
Made the dropdown text visible and disabled the info option

### LINKED ISSUE #3: Allow someone to be entered to multiple committees

### PR DESCRIPTION
**Pull request by hadiahmed098**

### CODE REVIEW

**Comment by hadiahmed098** [comment]
Should we make a similar change to the `view.php` page? It would be one more step from a usability standpoint but would be one less change we need to make on a yearly basis.

File: add.php [hunk 1]
```diff
@@ -114,7 +112,15 @@ function submit() {
             var email     = $("#email-input").val();
             var id        = $("#id-input").val();
             var enterer   = $("#enterer-input").val();
-            var committee = $("#id-committee").val();
+
+            var committees_total = document.getElementsByClassName("committees-checkbox");
+            var committee = "";
+            for(var i = 0; i < committees_total.length; ++i) {
+                if(committees_total[i].checked) {
+                    committee = committee.concat(", "  + committees_total[i].value);
```
Comment (grantg012) [inline add.php]: Dang really wish we could have python `", ".join(committee.value for committee in committees_total if committee.checked)`

**Comment by grantg012** [comment]
Looks good to me. My only question is should we wait for a dev interface on the server to test before merging (preferably rebase IMO)

File: add.php (hunk 1 above)
Comment (SwagatBhattacharyya) [reply add.php]: I like this option of being able to select multiple committees.  I had to manually limit the committees to 1 while uploading the CSV manually to make it consistent with what the interface would have input. For the sake of legislation and the like, it'd be great to have a "view by committee" so an officer can see their entire roster.

Comment (grantg012) [reply add.php]: If you go to the view page, you can search the table and it'll filter by your query. The only issue is that it doesn't let you search only in one column so the results can have some things you didn't intend. Ex: Someone named Bob Roven might show up when searching ROV. But this is a small issue

File: add.php [hunk 2]
```diff
@@ -162,7 +162,7 @@ function submit() {
                      $("#name-input").val("");
                      $("#email-input").val("");
                      $("#id-input").val("");
-                     $("#id-committee").val("None");
+                     $(".committees-checkbox").checked = false;
```
Comment (grantg012) [inline add.php]: I have no idea if this code works, but this section does need to be changed. Feel free to amend this commit if it's inevitably wrong.

File: add.php [hunk 3]
```diff
@@ -114,13 +117,10 @@ function submit() {
             var enterer   = $("#enterer-input").val();
 
             var committees_total = document.getElementsByClassName("committees-checkbox");
-            var committee = "";
-            for(var i = 0; i < committees_total.length; ++i) {
-                if(committees_total[i].checked) {
-                    committee = committee.concat(", "  + committees_total[i].value);
-                }
+            var committee = committees_total.filter(committee => committee.checked).join(", ");
```
Comment (hadiahmed098) [inline add.php]: Only reason I didn't do this last time is compatibility, how far back to we want to support non-ES6 (pre-2015) browsers?

Comment (grantg012) [reply add.php]: Nah I don't think we'll have anyone using old browsers

File: add.php (hunk 2 above)
Comment (grantg012) [reply add.php]: I realized, we can probably just not reset the committees checkboxes and call it a feature. Anyone entering multiple is likely to be doing so for the same committee

File: add.php (hunk 1 above)
Comment (hadiahmed098) [reply add.php]: I don't think that should be a huge problem, most committees are unlikely to be similar to names.

File: add.php (hunk 3 above)
Comment (hadiahmed098) [reply add.php]: Cool, I have one change to make but it looks good.

File: add.php (hunk 2 above)
Comment (hadiahmed098) [reply add.php]: Yeah probably, I can take care of that.

**Comment by grantg012** [comment]
It all _looks_ good to me. Have you tested everything?

**Comment by hadiahmed098** [comment]
Ah ha it does not work yet, I really dislike JS HTMLCollections vs Arrays.

**Comment by grantg012** [comment]
Lol well thank you for being the trooper to test it. Lmk once it's ready and I'll rebase/merge it

**Comment by hadiahmed098** [comment]
_Should_ be good to go, the committee selection properly gives a comma-separated string of all checked committees.

**Comment by hadiahmed098** [comment]
@grantg012 do you want to resolve conflicts and merge or should I?

**Comment by grantg012** [comment]
I got it. I think there's a more succinct way to do that concatenation too, so I'm going to mess around with the js/jQuery for a minute

**Comment by hadiahmed098** [comment]
Sounds good. jQuery might have a better way, but I am not familiar with it.

**Comment by grantg012** [comment]
Merged as f6eef65, e381ba0, 1ff3e25, and 005d912

**Pull request closed without merging**